src/text.ts

```diff
@@ -161,7 +161,7 @@ export const login = async ({ phone, code, logger, ip }: ILogin): Promise<string
         user.twilio = result
         await user.save()
       } catch (err) {
-        subLogger.error({err}, "impossible to fetch carrier")
+        subLogger.warn({err}, "impossible to fetch carrier")
```

Comment:
what's the effect of the code failing here? 

does this have any downstream effects that an unknowing person like me should know? 

Just thinking it would be a good practice to comment or log the effects on not being able to fetch the carrier.

Reply:
It's a non-critical failure. I've added a comment to make it clear.